src/Interpreters/DDLTask.cpp

```diff
@@ -103,7 +103,10 @@ String DDLLogEntry::toString() const
     }
 
     if (version >= OPENTELEMETRY_ENABLED_VERSION)
-        wb << "tracing: " << this->tracing_context;
+        wb << "tracing: " << tracing_context;
+
+    if (version >= PRESERVE_INITIAL_QUERY_ID_VERSION)
+        wb << "initial_query_id: " << initial_query_id;
```

Comment:
query_id is an arbitrary string, therefore you cannot serialize it without proper escaping or quoting.

Reply:
Nice catch! Fixed (and update the test).

BTW I don't see any purpose of allowing this, plus it breaks logging, maybe it worth to replace some symbols in query_id/initial_query_id.